pymongo/operations.py

```diff
@@ -508,3 +508,30 @@ def document(self) -> Dict[str, Any]:
         command.
         """
         return self.__document
+
+
+class SearchIndexModel:
+    """Represents a search index to create."""
+
+    __slots__ = "__document"
+
+    def __init__(self, definition: Mapping[str, Any], name: Optional[str] = None) -> None:
+        """Create a Search Index instance.
+
+        For use with :meth:`~pymongo.collection.Collection.create_search_indexes`.
```

Comment:
This link doesn't render for some reason. Might also be good to add:
```
... :meth:`~pymongo.collection.Collection.create_search_index` and
:meth:`~pymongo.collection.Collection.create_search_indexes`.
```

Reply:
Fixed